src/main.rs

```diff
@@ -104,17 +92,22 @@ fn gate(matches: (&str, Option<&ArgMatches>), mut driver: Box<Driver>, mots: Vec
     }
   }
   match matches {
-    ("down", _) => accelerator::down(&mut driver, &mots),
-    ("redo", _) => accelerator::redo(&mut driver, &mots),
-    ("reset", _) => accelerator::reset(&mut driver, &mots),
-    ("sub", Some(m)) => accelerator::shift(&mut driver, &mots, value_t!(m.value_of("n"), isize).unwrap_or(1) * -1),
-    ("shift", Some(m)) => accelerator::shift(&mut driver, &mots, value_t_or_exit!(m.value_of("n"), isize)),
-    ("goto", Some(m)) => accelerator::goto(&mut driver, &mots, value_t_or_exit!(m.value_of("n"), isize)),
-    _ => println!("Nothing to do!\nRe-run with --help for more information"),
+    ("down", _) => driver.down(&mots),
+    ("redo", _) => driver.redo(&mots),
+    ("reset", _) => driver.reset(&mots),
+    ("sub", Some(m)) => driver.shift(&mots, value_t!(m.value_of("n"), isize).unwrap_or(1) * -1),
+    ("shift", Some(m)) => driver.shift(&mots, value_t_or_exit!(m.value_of("n"), isize)),
+    ("goto", Some(m)) => driver.goto(&mots, value_t_or_exit!(m.value_of("n"), isize)),
+    _ => {
+      println!("Nothing to do!\nRe-run with --help for more information");
+      Ok(())
+    }
   }
 }
 
-fn get_driver(target: String) -> Box<Driver> { Box::new(drivers::default::Driver::new(target)) }
+fn get_driver(target: String) -> drivers::default::DefaultDriver {
```

Comment:
No, this is a temporary solution. I 100% understand why you used the `Box` now, but as long as we have one it shouldn't be needed.